tests/generate.py

```diff
@@ -1126,7 +1135,13 @@ def _revocation_test(
         intermediates_str: str = f"&[{int_a_str}, {int_b_str}]"
 
         def _write_revocation_test(*, owned: bool) -> None:
-            nonlocal crl_paths, expected_error, intermediates_str, test_name, ee_cert_path, root_cert_path
+            nonlocal \
```

Comment:
Unrelated.

Reply:
I think this diff is also the cause of the [one failed CI task](https://github.com/rustls/webpki/actions/runs/10025317179/job/27711164274?pr=272)